Breeding-Insight/bi-web
# [BI-1375] - Make GIDs clickable links

# Description
**Story:** [BI-1375 - Make GIDs clickable links](https://breedinginsight.atlassian.net/browse/BI-1375)

Added GermplasmLink.vue component for germplasm GID links
Updated GermplasmDetails.vue to make parent GIDs clickable links (but not the current germplasm GID as it would be redundant)
Updated GermplasmTable.vue to make all GIDs clickable links
Changed germplasmUUID in GermplasmDetails.vue to a computed property in order to enable load of new germplasm information on clicking a parent GID

# Dependencies
[bi-api/feature/BI-1375](https://github.com/Breeding-Insight/bi-api/pull/177)

# Testing
- Open Germplasm Table
- Check that each GID, Female Parent GID, and Male Parent GID in a row is a link and leads to the correct germplasm details page
- Open Germplasm Details Page for a germplasm with male and female parent
- Parent GIDs should be displayed and each should be a clickable link to their respective germplasm details page
- Open Germplasm Details Page for a germplasm with female parent only
- Female parent GID should be displayed and should be a clickable link to its respective germplasm details page
- Open Germplasm Details Page for a germplasm with no parents
- No parent GID links should be displayed

# Checklist:

- [X] I have run TAF: [_\<link to TAF run>_](https://github.com/Breeding-Insight/taf/actions/runs/2173893697)
